account_invoice_project/views/account_invoice_view.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+
+        <!-- customize invoice filters -->
+        <record model="ir.ui.view" id="view_account_invoice_filter">
+            <field name="name">account.invoice.select
+                (account_invoice_project)
+            </field>
+            <field name="model">account.invoice</field>
+            <field name="inherit_id" ref="account.view_account_invoice_filter" />
+            <field name="arch" type="xml">
+                <field name="partner_id" position="after">
+                    <field name="account_analytic_id" string="Project" />
```

Comment:
Seems that field must be a field name. It is possible to do it with force_domain (something like that) or maybe into a domain but I am afraid that's not possible for this case